tests/PactNet.Native.Tests/Messaging/MessageMiddlewareTests.cs

```diff
@@ -0,0 +1,99 @@
+//#if NET5_0_OR_GREATER
```

Comment:
These tests are commented out - not sure why.

Reply:
I have to put them back